changelog/changed-accessport-to-accessporttype.md

```diff
@@ -0,0 +1 @@
+Rename `AccessPort` to `AccessPortType`
```

Comment:
AccessPort(Type) has 3 changelog entries. A single added- should suffice.

Reply:
Well, the existing trait has been renamed, and a new enum type has been added.
So shouldn’t these two remain in distinct changelogs ?
The third entry for AccessPort addition is clearly a mistake though.

Reply:
Ah, so the order of operations was first rename, then add. Just by reading the changelog, I got the impression that you added AccessPort, then renamed it.